PlayerControls/sources/ContentControlsUIProps.swift

```diff
@@ -85,8 +85,9 @@ extension DefaultControlsViewController {
                 let isPlayerAbsent = props.player == nil
                 let isControlsViewHidden = !controlsViewVisible
                 let isAirPlayInactive = !(props.player?.item.playable?.airplay.isActive ?? false)
+                let isVoiceOverOff = !UIAccessibilityIsVoiceOverRunning()
```

Comment:
So our controls will not work without `UIAccessibilityIsVoiceOverRunning`?  I think this should not be here.